tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1976,11 +1992,22 @@ def adjust_hue(image, delta, name=None):
     Adjusted image(s), same shape and DType as `image`.
 
   Usage Example:
-    ```python
-    >> import tensorflow as tf
-    >> x = tf.random.normal(shape=(256, 256, 3))
-    >> tf.image.adjust_hue(x, 0.2)
-    ```
+    
+  >>> image = [[[1.0, 2.0, 3.0],[4.0, 5.0, 6.0]],...
+              [[7.0, 8.0, 9.0],[10.0, 11.0, 12.0]],...
+              [[13.0, 14.0, 15.0],[16.0, 17.0, 18.0]]]
+  >>> image = tf.constant(image)
+  >>> tf.image.adjust_hue(image, 0.2)
+  <tf.Tensor: shape=(3, 2, 3), dtype=float32, numpy=
+  array([[[ 2.3999996,  1.       ,  3.       ],
```

Comment:
Use fewer digits of precision (so replace 2.399996 with 2.3....); same below

Reply:
Okay. But before i will do that I will like to ask if it is proper to edit outputs? because those are  output array values not an input values

Reply:
> Can you use integer input values here?
> 
> Also, use a 2x2 matrix (here and everywhere else).

This can only be done with `tf.image.transpose()` and `tf.image.adjust_hue()`
all the others will raise ValueErrors